MaxLiu.md

```diff
@@ -1,31 +1,56 @@
-# [W04~05] Max Liu: 01/23 ~ 02/03
+# [W06~09] Max Liu: 02/06 ~ 03/03
 
 
 * [Fennec]
 
-  - Bug 1326291 - [RTL] Align the domain name in the site info panel to the right
-    - review+, landed
+  - Bug 1331995 - [RTL] On some occasions, the URL on the URL bar disappears
+    - landed
+
+
+  - Bug 1321981 - [RTL] Swiping gesture between Top Sites, Bookmarks and History in a New Tab is reversed
+  - Bug 1337440 - [RTL] Top Sites list's items in the Activity Stream should start from the right
```

Comment:
You should set assignee as yourself and document the below progress on Bugzilla (instead of here which gets overwritten next week).